Helper_Scripts/checks/email_search_dual_read_parity.py

```
@@ -51,20 +51,18 @@
 
-def _load_media_database_class():
-    try:
-        from tldw_Server_API.app.core.DB_Management.Media_DB_v2 import MediaDatabase as media_db_cls
-    except Exception:
-        print(
-            "tldw_Server_API not available; run from repo root or set PYTHONPATH.",
-            file=sys.stderr,
-        )
-        raise SystemExit(1) from None
-    return media_db_cls
+def _open_media_database(*, db_path: Path, client_id: str) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_open_media_database`

(ANN401)